Hello.java

```diff
@@ -1,7 +1,7 @@
 public class Hello {
 
        Hello(){
-           System.out.print("Hello, World!");
+           System.out.print("Hello, Kara!");
        }
        public static void main(String[] args) {
             Hello helloWorld = new Hello();
```

Comment:
I commented on this in the closed PR. Let me know if you can't find that comment.